Chinese/fighting.md

```diff
@@ -0,0 +1 @@
+金城盃打架事件後，台灣媒體立刻針對北大球員打人報導，報導內容多對北大球員的攻擊台藝大選手做描述，引發北大球員不滿，北大球員郭凱在微
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.